### DISCUSSION TITLE
Error during installation

### DISCUSSION THREAD

**Discussion by Aditish51** [theochem/AtomDB#29] [Q&A]
I am getting an error -ERROR: file:///mnt/c/Users/aditi/OneDrive/Desktop/OpenSource/Atomdb does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found. during installation

**Comment by msricher** [comment]
@maximilianvz could you help with this? I am unsure about it since you updated the install files.

**Comment by maximilianvz** [comment]
@Aditish51, I might need some more context from you about this installation error. In the screenshot I've attached below, I demonstrate how I installed the package locally, which works for me. Make sure you're doing `pip install -e .` from the top directory (it appears that you are, based on your error message), and that you have an environment with all of AtomDB's install dependencies and an updated version of `pip` to run the command (notice how I got an installation error before doing `conda activate env_ml_py38`). I do not understand why no `pyproject.toml` is being found if you're using the most up-to-date version of the repository.

<img width="1327" alt="Screenshot 2024-03-04 at 10 52 17 AM" src="https://github.com/theochem/AtomDB/assets/83951518/12de54a4-c613-49fc-a73a-ef05a4c6857f">

**Comment by msricher** [comment]
I am able to install AtomDB by first activating a venv and then running `pip install -e .` from the atomdb root directory.

**Comment by Aditish51** [comment]
@maximilianvz @msricher I will try to install it again.Thanks for the instructions.

**Reply by maximilianvz** [reply]
@Aditish51, a couple of things:

1. You have to do `pip install -e .` - don't miss the period. This specifies that you want to apply the command within your current working directory.
2. `conda activate env_ml_py38` only works on my computer, because this is the name of a virtual environment that I have installed locally. Are you familiar with virtual environments and how to install them (e.g., through Anaconda Navigator or command line)?

**Comment by Aditish51** [comment]
Yes, @maximilianvz I am familiar with installing virtual environment using command line @maximilianvz I installed it by the help of your instructions. Earlier I was using venv and was getting the error I mentioned above but then I did it using anaconda.Thanks for your help I will try to look for the issues.
![Screenshot from 2024-03-05 00-56-12](https://github.com/theochem/AtomDB/assets/150245832/47c7e5f6-b0db-46a4-84e2-3fc08a325f48)

**Comment by harshnayangithub** [comment]
Hello @Aditish51, Thanks for asking the installation doubt but I have installed AtomDB by CLI and it works perfectly fine without using anaconda.
![AtomDb](https://github.com/theochem/AtomDB/assets/126700987/b3cb62c3-0e8d-4da4-a329-e5abe884a79f)
